### DISCUSSION TITLE
Documents - improvements and translations

### DISCUSSION THREAD

**Discussion by miurahr** [miurahr/aqtinstall#592] [Ideas]
I've just pushed translations of documents in Japanese (my mother language).

As you know, we see more and more Qt software developers depends on aqtinstall for their CI runner, or distributions for their teams, next our target will be documentation.

I think we have enough contents of manuals at https://aqtinstall.readthedocs.io/en/latest/  as harvest which many contributors fosters.

Even it is enough volumes, there may be some places you can improve. So I'd like to call for contributions against documents.

1. Fix typo, update descriptions, check glossaries
2. Translations for your native languages

All contributions are welcome. Pull-Request is preferable way for us to propose changes, and also welcome raising issues of documents.

How to translate
---------------------

Please visit [docs/locale](https://github.com/miurahr/aqtinstall/tree/master/docs/locale) folder.
You can find a folder "pot" and "pot" files in it.
You can copy these "pot" files to your language folder, with renaming "*.po"

You can find a folder "ja/LC_MESSAGES"  that is an example.

Here is an example command to create 'de' translation.

```bash
$ cd aqtinstall  # where project root
$ python3 -m venv venv  # create virtualenv
$ source venv/bin/activate
$ pip install .[docs]  #  please not miss "." period
$ cd docs
$ sphinx-intl update -p locale/pot -l de   # generate po files for 'de'
```

then you can translate files with ``poedit`` or other translation tools to edit ``locale/de/LC_MESSAGES/*.po`` filies.
I used OmegaT, that is desktop translation assistant tool, on [a public team repository](https://github.com/miurahr/aqtinstall-docs-omegat-ja)

Check your translations
```bash
$ cd docs
$ make -e SPHINXOPTS="-D language='de'" html
```

![](https://www.sphinx-doc.org/en/master/_images/translation.svg)

Reference
-------------

* sphinx documentation tool manual https://www.sphinx-doc.org/en/master/usage/advanced/intl.html
* poedit: https://poedit.net/
* OmegaT: https://omegat.org/
* gettext: https://en.wikipedia.org/wiki/Gettext
* pootle: https://github.com/translate/pootle

**Comment by miurahr** [comment]
Now Japanese translation is provided. Do you have any preference for the other languages?

**Reply by pzhlkj6612** [reply]
> Do you have any preference for the other languages?

Maybe Chinese? I'm speaking Chinese. I can read both Traditional Chinese and Simplified Chinese, but correctly write Simplified Chinese only.
